PowerPointLabs/PowerPointLabs/Utils/FileDir.cs

```diff
@@ -205,6 +206,29 @@ public static void DeleteFile(string filePath)
             File.SetAttributes(filePath, FileAttributes.Normal);
             File.Delete(filePath);
         }
+
+        /// <summary>
+        /// Attempts to delete the file and does not delete the file if undeletable
+        /// </summary>
+        public static void TryDeleteFile(string filePath)
+        {
+            try
+            {
+                DeleteFile(filePath);
+            }
+            catch (Exception)
+            {
+            }

```

Comment:
Any reason why the catch clause is empty?

Reply:
added logging